packages/fast-element/src/directives/repeat.ts

```diff
@@ -5,14 +5,9 @@ import { DOM } from "../dom";
 import { Observable, IGetterInspector } from "../observation/observable";
 import { BindingDirective } from "./bind";
 import { ISyntheticView } from "../view";
-import {
-    synchronizeIndices,
-    applyMutationsToIndices,
-    ArrayObserver,
-    enableArrayObservation,
-    IndexMap,
-} from "../observation/array-observer";
 import { ISubscriber } from "../observation/subscriber-collection";
+import { ArrayObserver, enableArrayObservation } from "../observation/array-observer";
```

Comment:
Curious to our naming conventions?  Is there a difference in types between ArrayObserver vs enableArrayObservation?  Also, I like how you alphabetized the parameters.

Reply:
In this case `ArrayObserver` is a class, while` enableArrayObservation` is a function. Since the repeater and array observation are a large-ish feature, I've designed it to be tree shakeable. So, before you can use it, you need to import and call `enableArrayObservation`. This will happen automatically any time a `repeat` directive is used, so no need to worry about it in practical use.